lib/dnsimple/client.rb

```diff
@@ -173,8 +173,8 @@ def base_options
       options = {
           format:   :json,
           headers:  {
-            'Accept' => 'application/json',
-            'User-Agent' => user_agent
+              'Accept' => 'application/json',
```

Comment:
Ohh, guess this answers my question in https://github.com/aetrion/dnsimple-ruby/pull/93/files#r57599165...

I personally prefer 2 spaces too... (FWIW automatic formatting in vim with [vim-ruby](https://github.com/vim-ruby/vim-ruby) in place indents these using 2 spaces).